src/partials/_animal--lion.hbs

```diff
@@ -1,9 +1,8 @@
 <svg id="myDIV" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 1368 1422.1" width="500px" class="c-lion">
   <style>
-    .lion-0{fill:none;} .lion-1{fill:#F6BB60;} .lion-2{fill:#FFFFFF;} .lion-3{fill:#F1C586;} .lion-4{fill:#8A7967;} .lion-5{fill:#CFA16A;} .lion-6{fill:#231F20;} .lion-7{fill:#716558;} .lion-8{fill:#F6E7A1;}
+    .lion-1{fill:#F6BB60;} .lion-2{fill:#FFFFFF;} .lion-3{fill:#F1C586;} .lion-4{fill:#8A7967;} .lion-5{fill:#CFA16A;} .lion-6{fill:#231F20;} .lion-7{fill:#716558;} .lion-8{fill:#F6E7A1;}
```

Comment:
Since this is an inline SVG, this CSS could be a part of your Sass files if you’d like.